server/sonar-web/src/main/js/apps/projects/filters/LanguageFilter.js

```diff
@@ -39,6 +40,14 @@ export default class LanguageFilter extends React.Component {
     return sortBy(Object.keys(facet), [option => -facet[option]]);
   }
 
+  renderFooter = () => (
```

Comment:
Because in b4e1267 I moved `getFilterUrl` in an utils file and since the language searchbox is using it I thought it would be better to start this branch on top of `feature/greg/language_facet_search`